cloud-controller-manager/do/loadbalancers.go

```diff
@@ -96,6 +96,15 @@ const (
 
 var errLBNotFound = errors.New("loadbalancer not found")
 
+type errLBTimeout struct {
+	err  string
```

Comment:
`err` is conventionally used for `error` types. Might be a good idea to rename it to `message` or change the type to `error`.